app/views/your-psp/_stripe.njk

```diff
@@ -91,57 +92,67 @@
     </div>
   </div>
 
- {% else %}
- {% if  enableStripeOnboardingTaskList %}
+{% else %}
+  {% if enableStripeOnboardingTaskList %}
     <h1 class="govuk-heading-m govuk-!-margin-top-8">Information for Stripe</h1>
```

Comment:
We already have a `h1` on this page, and we should only have 1. If we're keeping this header, it should be a h2 and the current h2s should become h3.